Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel+VideoSegmentProcessor.swift

```diff
@@ -13,7 +13,9 @@ extension FaceLivenessDetectionViewModel: VideoSegmentProcessor {
         sendVideoEvent(data: chunk, videoEventTime: .zero)
         if !hasSentFinalVideoEvent,
            case .completedDisplayingFreshness = livenessState.state {
-            sendFinalVideoChunk(data: chunk, videoEventTime: .zero)
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.9) {
+                self.sendFinalVideoEvent()
+            }
```

Comment:
Does `sendFinalVideoEvent` send a network request on a different thread other than main? If not, I think we should wrap this in a background thread, like..
```suggestion
            DispatchQueue.main.asyncAfter(deadline: .now() + 0.9) {
              DispatchQueue.global(qos: .background).async {
                   self.sendFinalVideoEvent()
                }
            }
```

Reply:
The video processing is not on the main thread, I've updated the dispatch to be off of the main thread but with higher priority then background.